client/src/App.js

```diff
@@ -70,7 +59,7 @@ const App = () => {
           path="/admin"
           element={
             <PrivateRoute>
-              <DisplayProducts />
+              <Navigate to='/admin/products' />
```

Comment:
is this the right thing to do? will it be better to load the component here directly? instead of redirecting and then loading the component?

Reply:
It was display products previously someone changed it not me.

Reply:
Fixed it now